cmd/cve/generate.go

```diff
@@ -152,3 +221,34 @@ func gunzipFileAndComputeSHA256(filename string) (string, error) {
 	defer f.Close()
 	return gunzipAndComputeSHA256(f)
 }
+
+func gunzipFileToDisk(filename, dbpath string) error {
+	f, err := os.Open(filename)
+	if err != nil {
+		return fmt.Errorf("open file: %w", err)
+	}
+	defer f.Close()
+
+	gz, err := gzip.NewReader(f)
+	if err != nil {
+		return fmt.Errorf("new gzip reader: %w", err)
+	}
+	defer gz.Close()
+
+	filepath := filepath.Join(dbpath, strings.TrimSuffix(filepath.Base(filename), ".gz"))
+
+	out, err := os.Create(filepath)
+	if err != nil {
+		return fmt.Errorf("create file: %w", err)
+	}
+	defer out.Close()
+
+	// Using a maxBytes limit to prevent decompression bombs: gosec G110
+	maxBytes := 200 * 1024 * 1024 // 200MB
+	_, err = io.CopyN(out, gz, int64(maxBytes))
+	if err != nil && err != io.EOF {
+		return fmt.Errorf("copy file: %w", err)
```

Comment:
Double checking this fails if limit is reached. (So that we can detect the issue and bump it when it happens.)

Reply:
I added a panic here as it seems important enough